gatsby-config.js

```diff
@@ -27,8 +27,8 @@ module.exports = {
     {
       resolve: `gatsby-plugin-manifest`,
       options: {
-        name: 'gatsby-starter-contentful-typescript',
-        short_name: 'starter',
+        name: 'nodejs dev docs',
```

Comment:
```sugestion
        name: 'nodejs.dev',
```

Reply:
@MylesBorins Thank. I had typed the same first! 

Merged the commit suggestion.